docs/examples/userguide/wrapping_CPlusPlus/python_to_cpp.pyx

```diff
@@ -17,3 +19,12 @@ print(vect)  # [1, 3, 5, 7, 9]
 
 cdef vector[string] cpp_strings = b'It is a good shrubbery'.split()
 print(cpp_strings[1])   # b'is'
+
+# creates a python object, then convert it to C++ complex
+complex_val = 1+2j
+cdef complex[double] c_value1 = complex_val
```

Comment:
I can't use 

```
cdef complex[double] c_value1 = 1+2j
```

directly with the error

```
python_to_cpp.pyx:24:33: Cannot assign type 'double complex' to 'complex[double]'
```

the generated `.cpp` code has mentions of `complex double` but those only appear in commented code.